module-1/src/Components/MovieForm/MovieForm.tsx

```diff
@@ -30,71 +130,115 @@ export const MovieForm = ({ title, movie, closeCallback }: MovieFormProps) => {
           />
         </svg>
       </div>
+
       <h1 className="ml-5 mb-2 text-white text-2xl font-Montserrat">{title}</h1>
-      <form onSubmit={(e) => handleSubmit(e)}>
+      <form onSubmit={handleSubmit(onSubmit)}>
         <div className="grid grid-cols-2 gap-3 mx-5 pb-3">
           <div className=" grid col-auto">
             <label
               className="text-custom_pink font-Montserrat "
-              htmlFor="Title"
+              htmlFor="title"
             >
               Title
             </label>
             <input
+              {...register("title", { required: true })}
               className="bg-custom_light_gray text-white p-1"
               type="text"
-              name="Title"
-              id="Title"
-              value={movie?.movieName}
+              name="title"
+              id="title"
+              value={titleString}
```

Comment:
You should not manually control `value` of the input, it defies the idea of using a form library. `value`, in fact, will already be assigned by `register(...)` above, you simply override it. Same for all other inputs.

Reply:
fixed